test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -90,6 +92,29 @@ export class WebElementWrapper {
     return otherWebElements.map(e => this._wrap(e));
   }
 
+  private retryCount: number = 3;
+
+  private async retryCall(fn: Function, n: number = this.retryCount): Promise<any> {
```

Comment:
I personally find counting up an `attempt` variable, or counting down an `attemptsRemaining` variable a little easier to follow. Right now we're stopping when `n === 1`, which doesn't seem very intuitive.

```
fn(attempt = 1) {
  if (attempt < MAX_ATTEMPTS) {
    retry(attempt + 1)
  }
}
```

or 

```
fn(attemptsRemaining = MAX_RETRIES) {
  if (attemptsRemaining > 0) {
    retry(attemptsRemaining - 1)
  }
}
```